Bricks/util/test.cc

```diff
@@ -230,3 +233,10 @@ TEST(Util, SHA256) {
   EXPECT_EQ("a591a6d40bf420404a011733cfb7b190d62c65bf0bcda32b57b277d9ad9f146e",
             static_cast<std::string>(bricks::SHA256("Hello World")));
 }
+
+TEST(Util, RandomWithFixedSeed) {
+  EXPECT_EQ(114, bricks::random::RandomInt<int>(-100, 200));
```

Comment:
Looks like we'll have to remove the test :(
http://stackoverflow.com/questions/26663140/c-normal-distribution-gives-different-results-on-different-platforms